src/main/java/org/softc/armoryexpansion/common/integration/modsupport/MatterOverdriveIntegration.java

```diff
@@ -0,0 +1,51 @@
+package org.softc.armoryexpansion.common.integration.modsupport;
+
+import net.minecraft.block.Block;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import org.softc.armoryexpansion.ArmoryExpansion;
+import org.softc.armoryexpansion.common.integration.aelib.integration.JsonIntegration;
+
+@Mod(
+        modid = MatterOverdriveIntegration.MODID,
+        name = MatterOverdriveIntegration.NAME,
+        version = ArmoryExpansion.VERSION,
+        dependencies = MatterOverdriveIntegration.DEPENDENCIES
+)
+@Mod.EventBusSubscriber
+public class MatterOverdriveIntegration extends JsonIntegration {
+    static final String MODID = ArmoryExpansion.MODID + "-matteroverdrive";
+    static final String NAME = ArmoryExpansion.NAME + " - MatterOverdrive";
+    static final String DEPENDENCIES =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)